test/e2e_node/node_time_darwin.go

```diff
@@ -0,0 +1,42 @@
+// +build cgo,darwin
+
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_node
+
+import (
+	"time"
+)
+
+
+
+
+// GetNodeTime gets node boot time and current time.
+func GetNodeTime(initTime time.Time) (time.Time, time.Time, error) {
+	// Get node current time.
+	nodeTime := time.Now()
+
+	// TODO (aescobar): init time is a poor approximation for uptime, but darwin has little choice.
```

Comment:
Does [this](https://github.com/shirou/gopsutil/blob/master/host/host_darwin.go#L68-L97) seem reasonable for a more accurate Darwin uptime